src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,56 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        List<Manufacturer> manufacturers = List.of(
+                new Manufacturer("Dodge", "USA"),
+                new Manufacturer("Renault", "France"),
+                new Manufacturer("Peugeot", "France"),
+                new Manufacturer("BMW", "Germany"),
+                new Manufacturer("Mercedes", "Germany"),
+                new Manufacturer("Hyundai", "Korea"),
+                new Manufacturer("Subaru", "Japan")
+        );
+        ManufacturerService manufacturerService = (ManufacturerService)
+                injector.getInstance(ManufacturerService.class);
+
+        System.out.println("Manufacturers data will be displayed");
+        
+        manufacturers.forEach(manufacturerService::create);
+        System.out.println(manufacturerService.get(4L));
+        System.out.println(manufacturerService.update(
+                new Manufacturer(4L, "Volvo", "Sweden")));
+        manufacturerService.delete(5L);
```

Comment:
Do not hardcode ids! Otherwise, you are not able to run your code more than one time

Reply:
Sorry, now I got why is it so))